frontend/packages/kubevirt-plugin/src/components/vm-snapshots/vm-snapshots.tsx

```diff
@@ -64,6 +64,13 @@ export const VMSnapshotsTable: React.FC<VMSnapshotsTableProps> = ({
             sortField: 'status.readyToUse',
             transforms: [sortable],
           },
+          {
+            title: 'Last restored',
```

Comment:
@suomiy 
since the latest restore date is computed inside the row component (because one snapshot can have multiple restore objects) we can't sort the table by it. do you have any suggestions?

https://github.com/openshift/console/pull/6807/files#diff-d4bb250ed498bbf43eeee21385193b71R146

Reply:
@glekner you can use sortFunc prop

Reply:
done, hope it won't cause performance issues
https://github.com/openshift/console/pull/6807/files#diff-b2d4cf524cc534d555f71cccff72c6ecR172

Reply:
I am not a big fan of expecting that the core code should handle our CRDs. Wouldn't it be more straightforward to just resolve the time here and pass it as a data like we do in our other tables?